src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+.header {
+
+  // position: sticky;
+  // left: 0;
+  // right: 0;
+  top: 0;
+  display: flex;
+  height: $header-height;
+  width: 100%;
+  padding: 0 50px;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  margin-bottom: 25px;
```

Comment:
BEM-block shouldn't have margins

Reply:
Header isn't a BEM-block. Inner nav is a BEM, but it has no margins.